src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentRow.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 export default function AppointmentRow({
   children,
-  className,
+  className = '',
```

Comment:
Add 'undefined' to the class names if default is not defined.

Reply:
The initial intent was to encapsulate some of the complexity in the appointment list. So the 'AppointmentRow' component is just a wrapper with default class names defined for the common uses. Maybe a different property name is needed to convey that additional classes can be added but it is not required for this component.

Reply:
I see.